pkg/build/pipelines/ruby/build.yaml

```diff
@@ -0,0 +1,18 @@
+name: Build a ruby gem
+
+needs:
+  packages:
+    - busybox
+    - ca-certificates-bundle
+    - ruby-3.2
```

Comment:
This will require a package named `ruby-3.2` to exist in the package repo (ruling out Alpine-based pipelines using this workflow)

This will also mean that folks who want to build a gem for `ruby-3.1` (or just `ruby` for Alpine) won't be able to.

We could create variations of this pipeline like `ruby-3.2/build`, `ruby-3.1/build`, etc., but that gets messy.

We might just want to put this pipeline in wolfi-dev/os for now, and update the build system to set `--pipeline-dir=./pipelines/` so Wolfi's configs can use it. We'll still probably need a `ruby-3.1` variant, but at least the damage will be localized to Wolfi.

Reply:
Oh this seems to have already been addressed in https://github.com/chainguard-dev/melange/pull/262#discussion_r1105359773

Resolving this in favor of unresolving that.